app/views/images/index.html.erb

```diff
@@ -1,5 +1,4 @@
 <% title _("Images") %>
-<% title_actions display_link_if_authorized(_("New Image"), hash_for_new_compute_resource_image_path) %>
```

Comment:
It isn't - this is only used in the CR view which loads only the table via ajax.
I actually have a different PR turning this template into a partial with only the needed values - https://github.com/theforeman/foreman/pull/3365